src/GT.UI/Views/Inquiries/_InquiriesOverview.cshtml

```diff
@@ -11,14 +12,17 @@
 	<div>
 		<h2>Applications for @ViewData["JobTitle"]</h2>
 		<div>
-			@foreach (var item in Model)
-			{
-				<div class="card-body">
-					<h5>@Html.ActionLink(item.MessageTitle, "GetInquiry", new { id=item.Id }, new { @class="card-title stretched-link" })</h5>
-					<p class="card-text">@item.LinkedInLink</p>
-					@Html.ActionLink("Delete", "DeleteInquiry", new { id=item.Id }, new { @class="btn btn-danger"})
-				</div>
-			}
+      @if (ViewData["Inquiries"] is not null)
```

Comment:
Do we need null check on ViewData here?

Reply:
With the current implementation, yes, as ListingInquiryService.GetByListingIdAsync() can return null. The method could be refactored to return an empty list instead, but I opted to let it return null as it is in line with how other service methods work.